pallets/elections/src/lib.rs

```diff
@@ -56,6 +58,7 @@ pub mod pallet {
 
     #[pallet::config]
     pub trait Config: frame_system::Config {
+        type EraInfoProvider: EraInfoProvider;
```

Comment:
Shall we add some comments to new types and maybe existing ones? As pallet keeps growing. It seems it is standard Substarte way to comment `type`s for `pallet::Config`